src/vs/vscode.d.ts

```diff
@@ -6100,16 +6124,33 @@ declare module 'vscode' {
 		readonly selection: ReadonlyArray<T>;
 
 		/**
-		 * Reveal an element. By default revealed element is selected.
+		 * Event that is fired when the [selection](#TreeView.selection) has changed
+		 */
+		readonly onDidChangeSelection: Event<TreeViewSelectionChangeEvent<T>>;
+
+		/**
+		 * `true` if the [tree view](#TreeView) is visible otherwise `false`.
+		 */
+		readonly visible: boolean;
```

Comment:
This only means `visible` and not focused, right?

Reply:
Yes